spark-bigquery-pushdown/spark-3.2-bigquery-pushdown_2.13/src/main/scala/com/google/cloud/spark/bigquery/pushdowns/Spark32BigQueryPushdownPlan.scala

```diff
@@ -0,0 +1,28 @@
+package com.google.cloud.spark.bigquery.pushdowns
+
+import com.google.cloud.bigquery.connector.common.BigQueryConnectorException
+import com.google.cloud.spark.bigquery.pushdowns.SparkBigQueryPushdownUtil.doExecuteSparkPlan
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.expressions.Attribute
+import org.apache.spark.sql.execution.SparkPlan
+
+/** BigQueryPlan, with RDD defined by custom query. */
+case class Spark32BigQueryPushdownPlan(output: Seq[Attribute], rdd: RDD[InternalRow])
+  extends SparkPlan {
+
+  override def children: Seq[SparkPlan] = Nil
+
+  protected override def doExecute(): RDD[InternalRow] = {
+    doExecuteSparkPlan(output, rdd)
```

Comment:
Also couldn't find the implementation for this https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/master/spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/SparkBigQueryPushdownUtil.scala